src/oop2.py

```diff
@@ -3,12 +3,12 @@
 # Also change it so the num_wheels defaults to 4 if not specified when the
 # object is constructed.
 
-class GroundVehicle():
-    def __init__(self, num_wheels):
+class GroundVehicle:
+    def __init__(self, num_wheels=4):
         self.num_wheels = num_wheels
 
-    # TODO
-
+def drive(self):
```

Comment:
Indent here in order to have the def drive function be a method of the GroundVehicle class.